src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -20,6 +20,7 @@
     public static final String MESSAGE_ACTIVITIES_LISTED_OVERVIEW = "%1$d activities listed!";
     public static final String MESSAGE_ACCOMMODATIONS_LISTED_OVERVIEW = "%1$d accommodations listed!";
     public static final String MESSAGE_FRIENDS_LISTED_OVERVIEW = "%1$d friends listed!";
-    public static final String MESSAGE_TRAVELPLANS_LISTED_OVERVIEW = "%1$d travel plans listed!";
+    public static final String WRONG_DIRECTORY = "You are currently in the wishlist which can only find activity.\n"
+            + "Please goto to a travelplan to edit friend, activity or accommodation";
```

Comment:
Is this a clearer phrasing?
"Please go to your desired travelplan to edit friend, activity or accommodation, using the goto command.";